src/app/components/PageSection.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { Box, Flex, Stack, Text } from "@chakra-ui/react";
+
+interface PageSectionProps {
+  children?: React.ReactNode;
+  title: string;
+  bg?: string;
+  fullHeight?: boolean;
+  center?: boolean;
+}
+
+export const PageSection = (props: PageSectionProps) => {
+  return (
+    <Flex
+      flexDir="column"
+      w="100%"
+      bg={props.bg}
+      id={props.title}
+      m="auto"
+      alignItems={"center"}
```

Comment:
same curly braces issue. I think you can set your linter to fix this automatically